recipes/vulkan-loader/all/test_package/CMakeLists.txt

```diff
@@ -1,8 +1,8 @@
 cmake_minimum_required(VERSION 3.8)
 project(test_package LANGUAGES CXX)
 
-find_package(Vulkan REQUIRED)
+find_package(VulkanLoader REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
-target_link_libraries(${PROJECT_NAME} PRIVATE Vulkan::Vulkan)
+target_link_libraries(${PROJECT_NAME} PRIVATE Vulkan::Loader)
 target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
```

Comment:
Why cxx_std_11 btw? The Vulkan::Loader is a C99 library.

Reply:
Correct, but test_package.cpp uses C++11 (auto keyword).

Reply:
👍🏾